test/eChronicler_web/controllers/page_controller_test.exs

```diff
@@ -3,6 +3,8 @@ defmodule EChroniclerWeb.PageControllerTest do
 
   test "GET /", %{conn: conn} do
     conn = get(conn, "/")
-    assert html_response(conn, 200) =~ "Welcome to E-Chronicler!"
+    assert html_response(conn, 200) =~ "Journal Entries"
   end
+
```

Comment:
I want to put another test in here to check that the journal entries are in place, but I think I need to do some kind of buildup and pass the journal entry data into the HTML form first.

Reply:
Okay, I think I got this set up!